tools/create-cluster

```diff
@@ -2,28 +2,56 @@
 
 NAME="$1"
 if [ -z "$NAME" ]; then
-  echo "usage: create-cluster.sh <name>"
+  echo "usage: create-cluster <name>"
   exit 1
 fi
 
-CLUSTER_DIR="$HOME/clusters/${NAME}"
-if [ -d "$CLUSTER_DIR" ]; then
+CLUSTER_DIR="${HOME}/clusters/${NAME}"
+if [ -d "${CLUSTER_DIR}" ]; then
   echo "WARNING: cluster ${NAME} already exists at ${CLUSTER_DIR}"
+else
+  mkdir -p ${CLUSTER_DIR}
 fi
-
 # Generate a default SSH key if one doesn't exist
-SSH_KEY="$HOME/.ssh/id_rsa"
+SSH_KEY="${HOME}/.ssh/id_rsa"
 if [ ! -f $SSH_KEY ]; then
   ssh-keygen -t rsa -N "" -f $SSH_KEY
 fi
+export BASE_DOMAIN=openshift.testing
+export CLUSTER_NAME="${NAME}"
+export CLUSTER_ID=$(uuidgen --random)
+export PUB_SSH_KEY="${SSH_KEY}.pub"
+
+cat > "${CLUSTER_DIR}/install-config.yml" << EOF
```

Comment:
I saw you created a `create-install-config` script, did you mean to call it here?

Reply:
oops that was a previous commit, I _thought_ I had to separate the create install-config from the create cluster, but I was wrong (turns out it was failing bc I had to `mkdir -p ${CLUSTER_DIR}  : ).  I'll remove that.